src/node/node_state.h

```diff
@@ -1298,8 +1301,36 @@ namespace ccf
       // Open the service
       if (consensus->can_replicate())
       {
+        LOG_INFO_FMT(
+          "End of private recovery: attempting to write service open "
+          "transition in {}",
+          __func__);
```

Comment:
We have mostly used `__PRETTY_FUNCTION__` elsewhere, but this is standard and preferable.